inbox/react/migration-guide.mdx

```diff
@@ -0,0 +1,435 @@
+---
+title: "Migration Guide"
+sidebarTitle: "Migration Guide"
+description: "This guide outlines the key differences between the `@novu/notification-center` package and the new `@novu/react` package. Follow the steps below to migrate your application to the latest version."
+---
+
+## Getting Started
```

Comment:
The migration guide should include:
- intro section that will explain the purpose of the guide
- version summary: briefly introduce the new version and highlight the major changes; list breaking changes, v2 compatibility; mention new features, performance, and bundle improvements; 
- migration steps: installation section
- api changes: in this section list all that you have but with old/new way comparison
- styling changes

Reply:
cc @SokratisVidros